packages/docs/src/pages/en/features/theme.md

```diff
@@ -12,363 +12,199 @@ nav: Theme
-
-```js
-// src/plugins/vuetify.js
-
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-Vue.use(Vuetify)
-
-export default new Vuetify({
-  theme: { dark: true },
-})
-```
-
-When you designate a component as light or dark, all of its children will inherit and apply the same unless otherwise specified. You can manually turn **dark** on and off by changing `this.$vuetify.theme.dark` to **true** or **false**.
-
-## Customizing
-
-By default, Vuetify has a standard theme applied for all components.
-
-```js
-{
-  primary: '#1976D2',
-  secondary: '#424242',
-  accent: '#82B1FF',
-  error: '#FF5252',
-  info: '#2196F3',
-  success: '#4CAF50',
-  warning: '#FFC107',
-}
-```
-
-This can be easily changed. Simply pass a **theme** property to the Vuetify constructor. You can choose to modify all or only some of the theme properties, with the remaining inheriting from the default.
-
-```js
-// src/plugins/vuetify.js
-
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-const vuetify = new Vuetify({
-  theme: {
-    themes: {
-      light: {
-        primary: '#3f51b5',
-        secondary: '#b0bec5',
-        accent: '#8c9eff',
-        error: '#b71c1c',
-      },
-    },
-  },
-})
-```
-
-You can also use the pre-defined material colors.
+Vuetify comes with two themes pre-installed, `light` and `dark`. To set the default theme of your application, use the **defaultTheme** option.
 
 ```js
 // src/plugins/vuetify.js
+import { createApp } from 'vue'
+import { createVuetify } from 'vuetify'
 
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-import colors from 'vuetify/lib/util/colors'
-
-const vuetify = new Vuetify({
+export default createVuetify({
   theme: {
-    themes: {
-      light: {
-        primary: colors.purple,
-        secondary: colors.grey.darken1,
-        accent: colors.shades.black,
-        error: colors.red.accent3,
-      },
-      dark: {
-        primary: colors.blue.lighten3,
-      },
-    },
-  },
+    defaultTheme: 'dark'
+  }
 })
 ```
 
-By default, the theme service will use your application's primary color for **anchor** tags. You can override this by adding an anchor property to the theme:
-
-```js
-// src/plugins/vuetify.js
+Adding new themes is as easy as defining a new property in the **theme** object. A theme is a collection of colors and options that change the overall look and feel of your application. One of these options designates the theme as being either a **light** or **dark** variation. This makes it possible for Vuetify to implement Material Design concepts such as elevated surfaces having a lighter overlay color the higher up they are. Find out more about dark themes on the official [Material Design](https://material.io/design/color/dark-theme.html) page.
```

Comment:
What exactly are you referring to when you say `layouts and style elements`. I'm not sure this is the correct place to talk about SASS variables either. The **theme** functionality is separate from that, no? Changing theme will not change variables set by SASS since they are compile time.